src/components/EmojiPicker/EmojiPickerMenu/index.tsx

```diff
@@ -175,13 +175,13 @@ function EmojiPickerMenu({onEmojiSelected, activeEmoji}: EmojiPickerMenuProps, r
                 indexToSelect = 0;
             }
 
-            const item = filteredEmojis[indexToSelect];
+            const item = filteredEmojis.at(indexToSelect);
```

Comment:
Please make sure it doesn't break anything in case indexToSelect is -1, cause .at() accepts negative integers while [] not. See https://github.com/Expensify/App/pull/49534#discussion_r1772650621